packages/js/src/plugins/auctionHouseModule/operations/depositToBuyerAccount.ts

```diff
@@ -0,0 +1,202 @@
+import { ConfirmOptions, PublicKey } from '@solana/web3.js';
+import type { Metaplex } from '@/Metaplex';
+import { TransactionBuilder } from '@/utils';
+import {
+  createAuctioneerDepositInstruction,
+  createDepositInstruction,
+  DepositInstructionAccounts,
+} from '@metaplex-foundation/mpl-auction-house';
+import {
+  useOperation,
+  Operation,
+  OperationHandler,
+  Signer,
+  toPublicKey,
+  isSigner,
+  SolAmount,
+  SplTokenAmount,
+} from '@/types';
+import { SendAndConfirmTransactionResponse } from '../../rpcModule';
+import { findAssociatedTokenAccountPda } from '../../tokenModule';
+import { AuctionHouse } from '../models';
+import { findAuctioneerPda, findAuctionHouseBuyerEscrowPda } from '../pdas';
+import { AuctioneerAuthorityRequiredError } from '../errors';
+
+// -----------------
+// Operation
+// -----------------
+
+const Key = 'DepositToBuyerAccountOperation' as const;
+
+/**
+ * @group Operations
+ * @category Constructors
+ */
+export const depositToBuyerAccountOperation =
+  useOperation<DepositToBuyerAccountOperation>(Key);
+
+/**
+ * @group Operations
+ * @category Types
+ */
+export type DepositToBuyerAccountOperation = Operation<
+  typeof Key,
+  DepositToBuyerAccountInput,
+  DepositToBuyerAccountOutput
+>;
+
+/**
+ * @group Operations
+ * @category Inputs
+ */
+export type DepositToBuyerAccountInput = {
+  auctionHouse: Pick<
+    AuctionHouse,
+    | 'address'
+    | 'authorityAddress'
+    | 'hasAuctioneer'
+    | 'isNative'
+    | 'treasuryMint'
+    | 'feeAccountAddress'
+  >;
+  buyer?: PublicKey | Signer; // Default: identity
+  authority?: PublicKey | Signer; // Default: auctionHouse.authority
+  auctioneerAuthority?: Signer; // Use Auctioneer ix when provided
+  amount: SolAmount | SplTokenAmount;
```

Comment:
Could we start documenting these inputs, outputs and models as we implement them?

We can plan for backfilling all of that documentation later but it would be good if we could at least write documentation as we add new operations at the moment so we have less to do later.

Here's an example of a documented operation:

https://github.com/metaplex-foundation/js/blob/e4c74c22ef4fc7853821f72359f6d82b4bbf0029/packages/js/src/plugins/nftModule/operations/deleteNft.ts

Notice how the [operation constant](https://github.com/metaplex-foundation/js/blob/e4c74c22ef4fc7853821f72359f6d82b4bbf0029/packages/js/src/plugins/nftModule/operations/deleteNft.ts#L16-L28) and the [builder function](https://github.com/metaplex-foundation/js/blob/e4c74c22ef4fc7853821f72359f6d82b4bbf0029/packages/js/src/plugins/nftModule/operations/deleteNft.ts#L120-L132) have a small description and a little example of usage to help users get started.

Reply:
Thanks for the reference. Will do